app/models/concerns/project_media_creators.rb

```diff
@@ -122,7 +122,13 @@ def set_media_type
 
     if original_claim && original_claim.match?(/\A#{URI::DEFAULT_PARSER.make_regexp(['http', 'https'])}\z/)
       uri = URI.parse(original_claim)
-      content_type = Net::HTTP.get_response(uri)['content-type']
+      content_type = nil
+      begin
+        content_type = Net::HTTP.start(uri.host, uri.port, open_timeout: 3, read_timeout: 3, use_ssl: uri.scheme == 'https') { |http| http.request(Net::HTTP::Head.new(uri)) }['content-type']
```

Comment:
Suggestion: could we replace `http.request(Net::HTTP::Head.new(uri))` with `http.request_head(uri)`?

Testing locally it worked, I find it slightly easier to read.

Reply:
Ah, good catch! Yes, done!